lib/views/course_view/components/course_overview_section.dart

```diff
@@ -21,7 +23,7 @@ import 'package:gocast_mobile/views/utils/constants.dart';
 /// different titles, courses and onViewAll actions.
 class CourseSection extends StatelessWidget {
   final String sectionTitle;
-  final List<CourseModel>? courses;
+  final List<Course>? courses;
   final VoidCallback onViewAll;
 
   const CourseSection({
```

Comment:
Consider making the course-sections resize dynamically based on their content.
E.g., avoid the following, by either resizing them or adding some description (for example "_No courses found_") whenever `courses` are empty:
![image](https://github.com/TUM-Dev/gocast_mobile/assets/106114526/f79619c6-0bd3-4253-a3e1-0ea202167bc9)

Reply:
✅